docs/src/main/asciidoc/security.adoc

```diff
@@ -55,35 +55,39 @@ Please see xref:security-built-in-authentication.adoc#mutual-tls[Mutual TLS Auth
 
 === OpenID Connect
 
-`quarkus-oidc` extension provides a reactive, interoperable, multi-tenant enabled OpenID Connect adapter which supports `Bearer Token` and `Authorization Code Flow` authentication mechanisms.
+OpenID Connect (OIDC) is an open authentication protocol that enables client applications to verify the identity of a user when that user tries to connect to a secure server, such as HTTPS.
```

Comment:
```suggestion
OpenID Connect (OIDC) is an identity layer on top of the OAuth 2.0 protocol. It enables client applications to verify the identity of the user based on the authentication performed by the OIDC provider, as well as to obtain the basic information about the user.
```

Reply:
@sheilamjones I propose to reword similarly to how it is structured in the spec: https://openid.net/specs/openid-connect-core-1_0.html